### ISSUE TITLE
logically bound app images do not exist via bootc install

### ISSUE THREAD

**Issue by shi2wei3** [containers/bootc#762]
```
# cat ubi.image
[Image]
Image=registry.access.redhat.com/ubi9/ubi:latest

# cat nginx.container
[Container]
Image=docker.io/library/nginx:latest
GlobalArgs=--storage-opt=additionalimagestore=/usr/lib/bootc/storage
PublishPort=80:80

# cat Containerfile.lbi
FROM quay.io/centos-bootc/centos-bootc:stream10

COPY ./ubi.image /usr/share/containers/systemd/ubi.image
COPY ./nginx.container /usr/share/containers/systemd/nginx.container

RUN ln -s /usr/share/containers/systemd/ubi.image /usr/lib/bootc/bound-images.d/ubi.image && \
    ln -s /usr/share/containers/systemd/nginx.container /usr/lib/bootc/bound-images.d/nginx.container

# podman build -t centos-bootc -f Containerfile.lbi --from quay.io/centos-bootc/centos-bootc:stream10
# podman pull registry.access.redhat.com/ubi9/ubi:latest
# podman pull docker.io/library/nginx:latest
# podman run --tls-verify=false --rm --privileged --pid=host --security-opt label=type:unconfined_t -v /var/lib/containers:/var/lib/containers -v /dev:/dev localhost/centos-bootc \
    bootc install to-disk --skip-fetch-check  "$DEV" --filesystem xfs

VM# podman --storage-opt=additionalimagestore=/usr/lib/bootc/storage images
Error: configure storage: overlay: can't stat imageStore dir /usr/lib/bootc/storage: stat /usr/lib/bootc/storage: no such file or directory
VM# podman images
REPOSITORY                           TAG         IMAGE ID      CREATED      SIZE
registry.access.redhat.com/ubi9/ubi  latest      159a1e67312e  3 weeks ago  220 MB
docker.io/library/nginx              latest      900dca2a61f5  7 weeks ago  192 MB
VM# podman inspect registry.access.redhat.com/ubi9/ubi:latest
Error: faccessat /var/lib/containers/storage/overlay/624d0039f7ae2f740c806a420d084e7612569eaf651d6422ef5fd639fb640b78: no such file or directory
VM# podman inspect docker.io/library/nginx:latest
Error: faccessat /var/lib/containers/storage/overlay/db1e7509d4ebe12cc14d5740a97ebf48f900a37cd48bdc8b34c71f8f892ab38e: no such file or directory
```

Instead of using `bootc install` if I use `bootc switch` to the LBI bootc image, the bound app images exist in `/var/lib/containers/storage`, but `/usr/lib/bootc/storage` does not exists, so I don't understand what does it means by saying "bound images defined in the new bootc image will be automatically pulled into the bootc image storage" in https://github.com/containers/bootc/blob/main/docs/src/experimental-logically-bound-images.md#using-logically-bound-images

**Comment by ckyrouac** [comment]
```
# cat Containerfile.lbi
FROM quay.io/centos-bootc/centos-bootc:stream10
```

the latest bound images code that adds `bootc install` functionality and changes the storage location has not yet been included in a release. It should be available in 0.1.15.

**Comment by shi2wei3** [comment]
Good to know, thank you.
